src/Core/Usecase/Post/WebhookUpdatePost.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Ushahidi Platform Update Use Case
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Usecase\Post;
+
+use Ushahidi\Core\Entity;
+use Ushahidi\Core\Usecase\UpdateUsecase;
+
+class WebhookUpdatePost extends UpdateUsecase
```

Comment:
Why not just use the UpdatePostUsecase? Whats different in this instance?

Reply:
It uses a different update method since it's coming through the additional api